src/openrct2/object/DefaultObjects.cpp

```diff
@@ -11,65 +11,27 @@
 
 #include "Object.h"
 
-// clang-format off
-const std::string_view MinimumRequiredObjects[] = { "rct2.terrain_surface.grass", "rct2.terrain_edge.rock" };
+const std::array<std::string_view, 3> kMinimumRequiredObjects = {
```

Comment:
```suggestion
constexpr std::array<std::string_view, 3> kMinimumRequiredObjects = {
```

Reply:
That would require me to move them to the .h file, right?

Reply:
No, it just means it gets initialised at compile time instead of at runtime.

Reply:
Ah, I see. Updated it.